tasks/44_time-simulation/time-simulation.js

```diff
@@ -0,0 +1,18 @@
+export function SimTime() {
+  let simTime = 0;
+  let realTime = 0;
+  let speed = 1;
+
+  this.get = function() {
```

Comment:
Adding methods inside the constructor function has some drawbacks. Please use the prototype.